sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/LiveMetricConstants.cs

```diff
@@ -17,5 +17,17 @@ internal static class LiveMetricConstants
         internal const string ExceptionsPerSecondInstrumentName = "ExceptionsPerSecondLiveMetric";
         internal const string MemoryCommittedBytesInstrumentName = "CommittedBytesLiveMetric";
         internal const string ProcessorTimeInstrumentName = "ProcessorTimeBytesLiveMetric";
+
+        internal const string RequestDurationMetricIdValue = @"\ApplicationInsights\Request Duration";
+        internal const string RequestsMetricIdValue = @"\ApplicationInsights\Requests/Sec";
+        internal const string RequestsSucceededPerSecondMetricIdValue = @"\ApplicationInsights\Requests Succeeded/Sec";
```

Comment:
very nit: some of the conts have "PerSecond" in the name and others don't.

Reply:
RequestsMetricIdValue is the count of metrics. As the naming becomes complex avoided per second here. No strong opinion we could add to this one.